packages/@adobe/spectrum-css-temp/components/illustratedmessage/index.css

```diff
@@ -29,15 +29,18 @@ governing permissions and limitations under the License.
   text-align: center;
 
   .spectrum-IllustratedMessage-illustration {
+    order: 1;
     margin-bottom: var(--spectrum-illustrated-message-illustration-margin-bottom);
   }
 
   .spectrum-IllustratedMessage-heading {
+    order: 2;
     max-width: var(--spectrum-illustrated-message-heading-max-width);
     margin: var(--spectrum-illustrated-message-heading-margin);
   }
 
   .spectrum-IllustratedMessage-description {
+    order: 3;
```

Comment:
btw, since default for `order` is `0`, maybe good to do a `.spectrum-IllustratedMessage > * {order: 4;}`? and then just make all these immediate selectors to up their specificity

`> .spectrum-IllustratedMessage-description {order: 3;}`